ember-cli-flash/src/services/flash-messages.js

```diff
@@ -1,44 +1,42 @@
-/* eslint-disable ember/no-computed-properties-in-native-classes */
-import { equal, sort, mapBy } from '@ember/object/computed';
 import Service from '@ember/service';
 import { typeOf, isNone } from '@ember/utils';
 import { warn, assert } from '@ember/debug';
-import { set, get, setProperties, computed } from '@ember/object';
 import { classify } from '@ember/string';
-import { A as emberArray } from '@ember/array';
 import FlashMessage from '../flash/object';
 import objectWithout from '../utils/object-without';
 import { getOwner } from '@ember/application';
 import flashMessageOptions from '../utils/flash-message-options';
-import getWithDefault from '../utils/get-with-default';
 import { associateDestroyableChild } from '@ember/destroyable';
+import { tracked } from '@glimmer/tracking';
+import { registerDestructor } from '@ember/destroyable';
 
 export default class FlashMessagesService extends Service {
-  @(equal('queue.length', 0).readOnly())
-  isEmpty;
+  @tracked queue = [];
+
+  get arrangedQueue() {
+    return this.queue.sort(function (a, b) {
```

Comment:
is it ok to use sort here since sort mutates?

Reply:
Good point